frontend/src/app/services/game.ts

```diff
@@ -44,24 +44,37 @@ export class Game {
       }
     });
     this.mitigationStates.push(mitigationState);
+
+    while (this.lastDate < this.scenario.dates.rampUpEndDate) {
```

Comment:
I think this code should be outside the simulation, simulation core/class takes care of the simulation, mitigations and even the history enters as an argument to the simulation, and daily results would be output (stored/rendered) as needed.

But for now it does not matter, good we have it and can play it back somehow.